stubs/taint/core_privacy_security/django-filesystem-sinks.pysa

```diff
@@ -0,0 +1,12 @@
+def django.core.files.storage.FileSystemStorage.__init__(self, location: TaintSink[FileSystem_ReadWrite], base_url: TaintSink[FileSystem_ReadWrite], file_permissions_mode: TaintSink[FileSystem_ReadWrite], directory_permissions_mode: TaintSink[FileSystem_ReadWrite]): ...
+
+def django.core.files.storage.Storage.path(self, name: TaintSink[FileSystem_Other]): ...
```

Comment:
Same comment as `generate_filename` below. Can probably delete this model. If the generated path gets used in `open` as [the docs suggest it should](https://docs.djangoproject.com/en/1.11/ref/files/storage/#django.core.files.storage.Storage.path) we'll detect the issue at that point.